modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -10,23 +10,26 @@ class DisabilityCompensationController < BaseFormController
         FORM_NUMBER = '526'
         before_action { permit_scopes %w[claim.write] }
         before_action :verification_itf_expiration, only: %i[submit_form_526]
-        before_action :validate_526_payload, only: %i[submit_form_526]
         skip_before_action :validate_json_schema, only: %i[upload_supporting_documents]
         skip_before_action :verify_mvi, only: %i[submit_form_526 validate_form_526]
         skip_before_action :log_request, only: %i[validate_form_526]
 
         def submit_form_526
+          headers = auth_headers
```

Comment:
this seems like it's not doing anything unless I'm missing something

Reply:
There was a weird state in one of the tests that was expecting auth_headers to only be called once but it wasn't an explicit expect method so I couldn't quite figure out how to modify it in order to expect it to run twice so I went this route. Here is the actual test I'm talking about. 

https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb#L61-L70

Error is as follows and I can't for the life of me figure out how to expect 2 times instead of the one. 

```
1) Disability Claims  #526 should build the auth headers
     Failure/Error: expect(EVSS::DisabilityCompensationAuthHeaders).to receive(:new) { auth_header_stub }
     
       (EVSS::DisabilityCompensationAuthHeaders (class)).new(*(any args))
           expected: 1 time with any arguments
           received: 2 times with any arguments
     # ./modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb:65:in `block (5 levels) in <top (required)>'
```

Reply:
`expect(...).to receive(:whatever).twice` or however https://relishapp.com/rspec/rspec-mocks/docs/setting-constraints/receive-counts

Reply:
in this case maybe it should be an allow though

Reply:
I went with expect because it seemed more specific, you learn something every day and thanks for the assist, let me know what you think?